src/services/behaviors/index.ts

```diff
@@ -220,3 +220,22 @@ export const createActionHandler = (
   return (element: Element) =>
     onUpdate(null, action, element, { behaviorElement, custom: true });
 };
+
+/**
+ * Set the `ran-once` attribute to true.
+ */
+export const setRanOnce = (behaviorElement: Element) => {
+  behaviorElement.setAttribute('ran-once', 'true');
+};
+
+/**
+ * Checks if `once` is previously applied.
+ */
+export const isOncePreviouslyApplied = (behaviorElement: Element) => {
```

Comment:
let's type the returned value
```suggestion
export const isOncePreviouslyApplied = (behaviorElement: Element): boolean => {
```